### PR TITLE
Nicolas - SF Automation - Master LPD-35875

### PR DESCRIPTION
**Pull request by MicaelleSilvaa**
Ticket: https://liferay.atlassian.net/browse/LPD-35875

### CODE REVIEW

**Comment by MicaelleSilvaa** [comment]
@nicolas-sss

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by MicaelleSilvaa** [comment]
@nicolas-sss ready for review

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -3166,6 +3166,14 @@
...
 		"to": "CPTaxCategoryCreateDateComparator.getInstance(param#0#)"
 	},
+	{
+		"classNames": [
+			"IndexWriterHelper"
+		],
+		"from": "updateDocument(String paramName, long paramName, Document paramName, boolean paramName)",
+		"issueKey": "LPD-35875",
+		"to": "updateDocument(param#1#, param#2#)"
+	},
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:3176]: @MicaelleSilvaa  https://github.com/liferay-devtools/liferay-portal/blob/8e9196dc6656c42f6cc2514e87dbdb88a883344c/modules/util/source-formatter/src/main/resources/dependencies/replacements.json#L50

The `replacements.json` already has a pattern for a similar case in the utility class, can you check if doesn't also happen in the utility? If so, add in the `classNames` attribute.

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 2]
```diff
@@ -1190,7 +1198,7 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 
 		themeDisplay.getScopeGroupId();
 	}
-
+
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:1201]: Remove unnecessary tab.

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava [hunk 3]
```diff
@@ -1181,7 +1189,7 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 
 		themeDisplay.getPortletGroupId();
 	}
-
+
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava:1192]: Remove unnecessary tab.

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava [hunk 4]
```diff
@@ -1136,6 +1136,14 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		IndexWriterHelperUtil.updateDocument("test123", 123, null, null);
 	}
 
+	public void method(
+		String searchEngineId, long companyId, Document documents,
+		boolean commitImmediately) {
+
+		IndexWriterHelper.updateDocument(searchEngineId, companyId, documents, commitImmediately);
+		IndexWriterHelper.updateDocument("test123", 123, null, null);
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava:1144]: @MicaelleSilvaa you have added the reference at the bottom to use it to call the method, isn't a static method, so should use the variable.

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 5]
```diff
@@ -1145,6 +1145,14 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		IndexWriterHelperUtil.updateDocument("test123", 123, null, null);
 	}
 
+	public void method(
+		String searchEngineId, long companyId, Document documents,
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:1149]: Should be `document` in the singular

**Review by nicolas-sss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**